models/person/person.py

```diff
@@ -0,0 +1,17 @@
+class Person(object):
+    """The person class."""
+    id = 0
+
+    def __init__(self, first_name, last_name, wants_accomodation="N"):
+        """."""
+        type(self).id += 1
+        self.first_name = first_name.upper()
+        self.last_name = last_name.upper()
+        self.full_name = first_name + " " + last_name
+        self.wants_accomodation = wants_accomodation or "N"
```

Comment:
Since you already have a default value set for  ~`wants_accomodation`~ `wants_accommodation`, You don't need the `or "N"` part here.

It is fine as just 
```python
self.wants_accomodation = wants_accomodation
```